libraries/SPI/src/SPI.h

```diff
@@ -104,22 +114,22 @@ class SPIClass : public IOMaster {
 
   void setBitOrder(BitOrder order);
   void setDataMode(uint8_t uc_mode);
-  // void setClockDivider(uint8_t uc_div);
+  void setClockDivider(uint8_t div);
+
+  void _transfer(void *buf_out = NULL, void *buf_in = NULL, size_t count = 0);
```

Comment:
Not taking this change since '_transfer' is meant to be for internal use of the SPI ibrary